gcloud/bigtable/row.py

```diff
@@ -870,7 +870,7 @@ class _RegexFilter(RowFilter):
     """
 
     def __init__(self, regex):
-        self.regex = regex
+        self.regex = _to_bytes(regex)
```

Comment:
That breaks down if `to_pb` is called multiple times, since `_to_bytes` would be needed every time.